src/vuestic-theme/vuestic-components/vuestic-alert/VuesticAlert.vue

```diff
@@ -39,7 +44,8 @@ export default {
   },
   methods: {
     hide () {
-      this.hidden = true
+      this.hidden = true;
+      this.onClose();
```

Comment:
Maybe add `@hide` or `@close` event?
cc @asvae

Reply:
This should be done with events, indeed.